package.json

```diff
@@ -87,6 +87,8 @@
     "url-loader": "^0.5.7",
     "webpack": "1.12.14",
     "webpack-dev-middleware": "^1.2.0",
-    "webpack-dev-server": "^1.14.1"
+    "webpack-dev-server": "^1.14.1",
+    "react-virtualized": "^7.3.1",
+    "react-addons-shallow-compare": "^15.1.0"
```

Comment:
Thought I wouldn't remove it just yet in case someone wants to switch back to fixed-data-table ?